include/extractor/profile_properties.hpp

```diff
@@ -30,10 +30,18 @@ struct ProfileProperties
         traffic_signal_penalty = boost::numeric_cast<int>(traffic_signal_penalty_ * 10.);
     }
 
+    double GetMaxSpeedForMapMatching() const { return max_speed_for_map_matching; } // /3.6 converts kmph to m/s
+
+    void SetMaxSpeedForMapMatching(const double max_speed_for_map_matching_) 
+    {
+        max_speed_for_map_matching = boost::numeric_cast<int>(max_speed_for_map_matching_);  // *3.6 converts  m/s to kmph
```

Comment:
Neither here not a couple of lines above is a 3.6 multiplication - comments off?